docs/guide/options.md

```diff
@@ -69,3 +68,22 @@ const chart = new Chart('id', {
 | `onZoom` | `{chart}` | Called while the chart is being zoomed
 | `onZoomComplete` | `{chart}` | Called once zooming is completed
 | `onZoomRejected` | `{chart,event}` | Called when zoom is rejected due to missing modifier key. `event` is the a [hammer event](https://hammerjs.github.io/api#event-object) that failed
+
+## Range
+
+Range options define the limits for scale pan and zoom.
+
+### Range options
+
+| Name | Type | Description
+| ---- | -----| -----------
+| `x` | [`ScaleRange`](#scale-range) | Range limits for x-axis
+| `y` | [`ScaleRange`](#scale-range) | Range limits for x-axis
+
+#### Scale Range
+
+| Name | Type | Description
+| ---- | -----| -----------
+| `min` | `number` | Minimun allowed value for scale.min
+| `max` | `number` | Maximum allowed value for scale.max
+| `range` | `number` | Minimum allowed range (max - min). This defines the max zoom level.
```

Comment:
Maybe it's just me but `range.range` naming is a bit confusing, and doesn't reflect that defines a limit. ^^ What do you think about `minRange` or `limit`... ?

Reply:
Changed the option key to `limits`, so `limits.min`, `limits.max` and `limits.range`. What do you think?

Reply:
Maybe it should still be `minRange` to leave room for possible `maxRange`.

Reply:
As you want, I think that `limits.min`, `limits.max` and `limits.range` is nice! Thanks!